ambry-commons/src/test/java/com.github.ambry.commons/BlobIdTest.java

```diff
@@ -593,6 +555,7 @@ private void assertBlobIdFieldValues(short version, BlobId blobId, BlobIdType ty
         assertNull("Expected null blobDataType in blobId", blobId.getBlobDataType());
         break;
       case BLOB_ID_V5:
+      case BLOB_ID_V6:
```

Comment:
there is a function named `buildBadBlobId()` that has a switch case. I might have missed the switch case change in that function but if you haven't added it, please take a look

Reply:
Added v6 support in this test